### ISSUE TITLE
InputClock/OutputClock missing in causality enumeration?

### ISSUE THREAD

**Issue by clagms** [modelica/fmi-standard#914]
In section "Definition of Model Variables (ModelVariables)", in the table under the text "The attributes of variables are:", the causality attributed has an inconsistent description of the different values allowed.

There are two issues.

The first issue is that the text is incorrectly specified, as it fails to describe the `inputClock` and `outputClock` values:
```
[[inputClock,`input clock`]]
- `input`: the variable defines an <<inputClock>> that is controlled by the master.
Variability must be <<clock>>.
Only a variable of type `<Clock>` can have this <<causality>>.

[[outputClock,`output clock`]]
- `output`: the variable defines an <<outputClock>> that is controlled by the FMU.
Variability must be <<clock>>.
Only a variable of type `<Clock>` can have this <<causality>>.
```
In the above, `input` (resp. `output`) should be `inputClock` (resp. `outputClock`), no?

The second issue is that looking at the XSD, it seems to be missing the values `inputClock` and `outputClock` in the causality enumeration.

I don't know what exactly is the solution here, but maybe at some point there was an `inputClock` causality that was later replaced by just using clock variable with causality `input`?

**Comment by clagms** [comment]
According to @IZacharias , we must move those sentences to the input/output attributes. Specify that these are applicable to variables with variability=clock.

**Comment by IZacharias** [comment]
Has been fixed with #938 . Ticket can be closed.
